File: pkg/codegen/testing/test/testdata/using-dashes-pp/go/using-dashes.go
package main

import (
	usingDashes "github.com/pulumi/pulumi-using-dashes/sdk/go/using-dashes"
	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
)

func main() {
	pulumi.Run(func(ctx *pulumi.Context) error {
		_, err := usingDashes.NewDash(ctx, "main", &usingDashes.DashArgs{
			Stack: pulumi.String("dev"),
		})
		if err != nil {
			return err
		}
		return nil
	})
}

File: pkg/codegen/dotnet/testdata/using-dashes-pp/using-dashes.cs
using System.Collections.Generic;
using System.Linq;
using Pulumi;
using UsingDashes = Pulumi.UsingDashes;

return await Deployment.RunAsync(() => 
{
    var main = new UsingDashes.Dash("main", new()
    {
        Stack = "dev",
    });

});

